tests/docker/Dockerfile

```diff
@@ -21,7 +21,7 @@ ENV MIRROR="http://apache.cs.utah.edu/"
 
 ADD ssh /root/.ssh
 RUN chmod 600 /root/.ssh/id_rsa
-RUN apt update && apt install -y unzip wget curl jq coreutils openssh-server net-tools vim openjdk-8-jdk python-pip python-dev libffi-dev libssl-dev
+RUN apt update && apt install -y unzip wget curl jq coreutils openssh-server net-tools vim python-pip python-dev libffi-dev libssl-dev
```

Comment:
@ewencp Since we start with `openjdk:8` image this is not necessary. (see line 16: FROM openjdk:8). And this change will not break anything.